src/main/java/bowling/domain/NormalFrame.java

```diff
@@ -0,0 +1,51 @@
+package bowling.domain;
+
+public class NormalFrame implements Frame {
+    public static final int MAX_TRY_COUNT = 2;
+    private BowlResult bowlResult;
+    private int trying;
+
+    private NormalFrame() {
+        this.bowlResult = new BowlResult();
+        this.trying = 0;
+    }
+
+    private NormalFrame(BowlResult bowlResult, int trying) {
+        this.bowlResult = bowlResult;
+        this.trying = trying;
+    }
+
+    public static Frame newInstance() {
+        return new NormalFrame();
+    }
```

Comment:
`firstBowl()`메서드에서 새로운 `Frame` instance를 만드는 구조라면, `newInstance()` 에서는 동일한 인스턴스를 전달해도 괜찮지 않을까요?
혹은 초기 인스턴스 필드를 public으로 노출할 수도 있겠구요.

`BowlResult`도 동일한 구조로 동작하는데 함께 고민해보셨으면 해요.

```suggestion
    private static final Frame INSTANCE = new NormalFrame(new BowlResult(), 0);
    public static Frame newInstance() {
        return INSTANCE;
    }
```